src/util.js

```diff
@@ -30,7 +30,8 @@ const decoder = new cbor.Decoder({
       val = val.slice(1)
       return {'/': val}
     }
-  }
+  },
+  size: 1e+6 /* one megabyte */
```

Comment:
Looking at the implementation, it tries to use power of two. So it make make sense to write is as `size: 1024 * 1024 * 1024` (which I find easier to read, but that's a matter of taste).